jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/RetainableByteBuffer.java

```diff
@@ -1445,7 +1445,7 @@ public DynamicCapacity(ByteBufferPool pool, boolean direct, long maxSize, int ag
 
         private DynamicCapacity(List<RetainableByteBuffer> buffers, ByteBufferPool.Sized pool, long maxSize, int minRetainSize)
         {
-            super();
+            super(new ReferenceCounter()); // Make sure the wrapped retainable is a ReferenceCounter.
```

Comment:
The comment tells what the code does, so it can be removed.
Actually the previous code was doing the same, so I'd restore the previous code.

Reply:
or remove the `super()` call entirely?

Reply:
Done.